db_test.go

```diff
@@ -32,17 +46,37 @@ func setupDB(t *testing.T) *surrealdb.DB {
 	return db
 }
 
-func openConnection(t *testing.T) *surrealdb.DB {
+func setupDBOptions(t *testing.T) *surrealdb.DB {
+	db := openConnectionWithOptions(t)
+	_ = signin(t, db)
+	_, err := db.Use("test", "test")
+	require.NoError(t, err)
+	return db
+}
+
+func openConnectionWithOptions(t *testing.T) *surrealdb.DB {
```

Comment:
You can make the testing suite instance a struct property for this suite of tests. That way you could invoke such functions without passing in `*testing.T`